.stylelintrc.json

```diff
@@ -1,97 +1,38 @@
 {
   "extends": "stylelint-config-standard-scss",
   "rules": {
```

Comment:
Should we include a comment/description somewhere of the three blocks, I think that makes sense.

Reply:
No, we can't. I tried. I'd like it. But I tried several hacky ways. It's impossible.

JSON does not support comments. And Stylelint doesn't support arbitrary bogus rules.